addons/redis/redis-cluster-scripts/redis-cluster-manage.sh

```diff
@@ -443,16 +453,19 @@ initialize_redis_cluster() {
   for primary_pod_name in "${!initialize_redis_cluster_primary_nodes[@]}"; do
     primary_nodes+="${initialize_redis_cluster_primary_nodes[$primary_pod_name]} "
   done
+  set +x
   if [ -z "$REDIS_DEFAULT_PASSWORD" ]; then
+    echo "initialize command: redis-cli --cluster create $primary_nodes --cluster-yes"
```

Comment:
what's the "echo" command used for if debug mode is disabled?

Reply:
> what's the "echo" command used for if debug mode is disabled?

In the shell, the set -x option tells Bash to print each command to the standard output before executing it, while set +x temporarily disables command printing. The echo command is used to output some auxiliary debug information even when set +x has turned off command printing.

Reply:
I see, so the script will output a lot more logs than before because "set -x" is set?